cherab/tools/observers/bolometry.py

```diff
@@ -276,229 +227,151 @@ def __init__(self, detector_id, centre_point, basis_x, dx, basis_y, dy, slit,
-            state['volume_power_sensitivity'] = self._volume_power_sensitivity.__getstate__()
-            state['etendue'] = self._etendue
-            state['etendue_error'] = self._etendue_error
-
-        return state
+            raise ValueError("The units argument of BolometerFoil must be one of 'Power' or 'Radiance'.")
+
+        super().__init__([slit.target], targetted_path_prob=1.0,
+                         pipelines=[pipeline],
+                         pixel_samples=1000, x_width=dx, y_width=dy, spectral_bins=1, quiet=True,
+                         parent=parent, transform=translation * rotation, name=detector_id)
+
+    def __repr__(self):
+        """Returns a string representation of this BolometerFoil object."""
+        return "<BolometerFoil - " + self.name + ">"
 
     @property
     def centre_point(self):
-        return self._centre_point
+        return Point3D(0, 0, 0).transform(self.to_root())
 
     @property
-    def normal_vec(self):
-        return self._normal_vec
+    def normal_vector(self):
+        return Vector3D(0, 0, 1).transform(self.to_root())
 
     @property
     def basis_x(self):
-        return self._basis_x
+        return Vector3D(1, 0, 0).transform(self.to_root())
 
     @property
     def basis_y(self):
-        return self._basis_y
+        return Vector3D(0, 1, 0).transform(self.to_root())
+
+    @property
+    def sightline_vector(self):
+        return self.centre_point.vector_to(self._slit.centre_point)
 
     @property
     def slit(self):
         return self._slit
 
-    @property
-    def los_radiance_sensitivity(self):
-        if self._los_radiance_sensitivity is None:
-            raise ValueError("The sensitivity of this BolometerFoil has not yet been calculated.")
-        return self._los_radiance_sensitivity
+    def as_sightline(self):
 
-    @property
-    def volume_power_sensitivity(self):
-        if self._volume_power_sensitivity is None:
-            raise ValueError("The sensitivity of this BolometerFoil has not yet been calculated.")
-        return self._volume_power_sensitivity
+        if self.units == "Power":
+            pipeline = PowerPipeline0D(accumulate=False)
+        elif self.units == "Radiance":
+            pipeline = RadiancePipeline0D(accumulate=False)
+        else:
+            raise ValueError("The units argument of BolometerFoil must be one of 'Power' or 'Radiance'.")
 
-    @property
-    def volume_radiance_sensitivity(self):
-        if self._volume_radiance_sensitivity is None:
-            raise ValueError("The sensitivity of this BolometerFoil has not yet been calculated.")
-        return self._volume_radiance_sensitivity
+        los_observer = SightLine(pipelines=[pipeline], pixel_samples=1, quiet=True,
+                                 parent=self, name=self.name, transform=self.transform)
+        los_observer.render_engine = self.render_engine
```

Comment:
There is no way to specify `self.render_engine` in init. So presumably we should set BolometerFoil.render_engine before calling `as_sightline`, or manually set `los_observer.render_engine` after calling this function. This should be made clear in the documentation.

Reply:
Yes the intention here is to bring this class more inline with the conventions of our other Observers. Raysect observers never set the render engine through the init, its always something like:
```
observer = PinholeCamera()
observer.render_engine = SerialEngine()
```

So I think this code works as expected because the SighLline observer you request will just copy all the appropriate settings from the parent BolometerFoil.